rpcs3/Emu/VFS.cpp

```diff
@@ -462,3 +473,31 @@ bool vfs::host::rename(const std::string& from, const std::string& to, bool over
 
 	return true;
 }
+
+bool vfs::host::unlink(const std::string& path)
+{
+#ifdef _WIN32
+	if (path.size() < 2 || reinterpret_cast<const u16&>(path.front()) != "//"_u16)
+	{
+		// Rename to special dummy name which will be ignored by VFS (but opened file handles can still read or write it)
```

Comment:
Aren't POSIX unlink semantics supported by newer version of Win10 making this workaround unnecessary (though it's still opt-in)?

Reply:
But that's only on newer Windows 10, RPCS3 is made for Win7+

Reply:
Haven't seen any new things for this in WinAPI, so it's either undocumented or worse (not supposed to be used by windows programs at all).

Reply:
`FILE_DISPOSITION_POSIX_SEMANTICS` flag seems to be documented [here](https://docs.microsoft.com/en-us/windows-hardware/drivers/ddi/content/ntddk/ns-ntddk-_file_disposition_information_ex). Available since Win 10 RS1 so it's new and not many are aware about it. Here it's used in [llfio](https://github.com/ned14/llfio/blob/5f576c797bfbadfbcc9092ce7e1b98089156eebc/include/llfio/v2.0/detail/impl/windows/fs_handle.ipp#L184).